### ISSUE TITLE
[Clang] constexpr support for __builtin_convertvector and __builtin_shufflevector

### ISSUE THREAD

**Issue by philnik777** [llvm/llvm-project#107985]
LanguageExtensions.rst claims that `__builtin_shufflevector` and `__builtin_convertvector` can be constant evaluated, but that's not reflected inside `Butiltins.td`.

**Comment by zwuis** [comment]
This is part of #46593.

**Comment by RKSimon** [comment]
This was fixed by #76615

**Comment by philnik777** [comment]
@RKSimon No, it hasn't: https://godbolt.org/z/c3bTe5Y8E. This report is about the inconsistency that it's documented as such inside `LanguageExtensions.rst` but not actually marked as such, i.e. a `Constexpr` attribute is missing from `Builtins.td`.
